packages/typespec-ts/test/nonBrandedIntegration/modular/generated/models/usage/src/UsageClient.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Do we have a way to replace this?

Reply:
Currently no, we hard coded this. But we could replace them, any suggestions or default copyright info for non branded?

Reply:
I removed the line for `// Copyright (c) Microsoft Corporation.` in non-branded.